app/vmalert/remotewrite/remotewrite.go

```diff
@@ -183,6 +183,7 @@ func (c *Client) run(ctx context.Context) {
 var (
 	sentRows            = metrics.NewCounter(`vmalert_remotewrite_sent_rows_total`)
 	sentBytes           = metrics.NewCounter(`vmalert_remotewrite_sent_bytes_total`)
+	sentDuration        = metrics.NewFloatCounter(`vmalert_remotewrite_sent_duration_seconds_total`)
```

Comment:
I think we should expose additionally `vmalert_remotewrite_queues` metric (as we do in vmagent).
Since each worker sends data independently the send duration might be skewed (updated by multiple workers). So in order to get average saturation we should do the following math: `rate(send_duration_seconds) / remotewrite_queues`.
See more about this reasoning here https://github.com/VictoriaMetrics/VictoriaMetrics/commit/27f1c65074b1daafd07359b98f7b89b4f798a786